test/uk/gov/hmrc/childcarecalculatorfrontend/views/ChildrenAgeGroupsViewSpec.scala

```diff
@@ -46,9 +46,9 @@ class ChildrenAgeGroupsViewSpec extends NewViewBehaviours with NewCheckboxViewBe
     mockView(form, NormalMode)(fakeRequest, messages)
 
   "ChildrenAgeGroupsView" must {
-    behave like normalPage(createView, messageKeyPrefix)
+    behave like normalPage(() => createView(), messageKeyPrefix)
 
-    behave like pageWithBackLink(createView)
+    behave like pageWithBackLink(() => createView())
```

Comment:
What's the purpose of this change? The previous version seems to be correct.

Reply:
Okay, now I see the root cause.
Changing to `createView()` won't work, but you could try changing `def createView()` in `NewCheckboxViewBehaviours[A]` (line 27) into:
```
val createView: () => Html = () => createView(form)
```

After that you should be able to stay with just passing the `val`s name (`createView`).

Reply:
Done 👍